src/dodal/beamlines/p38.py

```diff
@@ -318,3 +319,17 @@ def linkam(
         wait_for_connection,
         fake_with_ophyd_sim,
     )
+
+
+def high_pressure_xray_cell(
+    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
+) -> PressureJumpCell:
+    return device_instantiation(
+        PressureJumpCell,
+        "high_pressure_xray_cell",
+        "-EA-HPXC-01:",
+        wait_for_connection,
+        fake_with_ophyd_sim,
+        adc1_prefix=f"{BeamlinePrefix(BL).beamline_prefix}-EA-ADC-01:",
+        adc2_prefix=f"{BeamlinePrefix(BL).beamline_prefix}-EA-ADC-02:",
```

Comment:
should:  extract the value the the function body

Reply:
Moved the adc numbering definition into PressureJumpCell's constructor in 2face7c257b63e3b682f08fbf87266fe1ea7e12c. This would be improved by correcting the PV naming in the device driver itself so the adc would be a sub-section of cell prefix, but I suggest testing first with the unmodified driver.